### PR TITLE
Add campus climate survey visual

### PR DESCRIPTION
**Pull request by Nik4002**
## How was the functionality tested and verified?
All steps should be completed in the order presented, and you should not move on to the next step until the previous is completed.
- [x] The title of the `html` page is the same as the chart.
- [x] The `meta_data.json` file is filled in correctly (the description is a complete sentence, the title is the same as the visual).
- [x] The chart and its associated pages have been minified using the `npm run process-visual` command.
- [x] The visual is styled similar to other charts (centered title/subtitle, no legends unless absolutely necessary, etc.)
- [ ] This visual was approved by the lead developer.
- [x] The visual was approved by the story's stakeholders.

Screen Shot 2024-02-28 at 9 27 50 PM

### CODE REVIEW

File: 2024/campus-climate-survey-by-university/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    },
+
+    caption: {
+        text: '*Stanford reported a 29-31% response rate from students.',
+        align: 'center'
+    },
+
+    xAxis: {
+        categories: ['UChicago', 'MIT', 'Stanford*', 'Brown'],
+        crosshair: true
```
Comment (michplunkett) [inline 2024/campus-climate-survey-by-university/main.js]: Do we need this value?

File: 2024/campus-climate-survey-by-university/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    },
+
+    tooltip: {
+        valueSuffix: '%'
+    },
+
+    series: [
+        {
+            name: 'Students',
+            type: 'column',
```
Comment (michplunkett) [inline 2024/campus-climate-survey-by-university/main.js]: Since you define the chart `type` above, I believe you can remove the series `type` keys.

**Review by michplunkett — CHANGES_REQUESTED** [review]
Smöl changes.

File: 2024/campus-climate-survey-by-university/main.js (hunk 1 above)
Comment (Nik4002) [reply 2024/campus-climate-survey-by-university/main.js]: Which value? Crosshair?

File: 2024/campus-climate-survey-by-university/main.js (hunk 2 above)
Comment (Nik4002) [reply 2024/campus-climate-survey-by-university/main.js]: Yes, youre right, I added those to differentiate the series from the error bar but now that that's gone, I can remove it again.

**Review by michplunkett — APPROVED** [review]
LGTM!

**Pull request merged**